pkg/binutils/processes.go

```diff
@@ -26,6 +29,12 @@ import (
 // gRPCTimeout is a common error message if the gRPC server can't be reached
 var gRPCTimeout = errors.New("Timed out trying to contact backend controller, it is most probably not running.")
 
+var latestRunDir string
+
+func GetLatestRunDir() string {
```

Comment:
is it going to be available in a subsecuent deploy or start command? because the server will continue to be alive.
It seems the idea of putting the ANR server logs in the same directory as the ANR local network logs is ok for debugging purposes. 
But as the server lifetime exceeds that of a ANR local network (you can deploy, then stop, then deploy, stop, start, all using the same server).
The directory struct seems to be like this:
```
server directory
- ANR directory 1
- ...
- ANR directoryN
```
What do you think on this?

Reply:
Anyway GetLatestRunDir maybe can be renamed to something related to backend or server rundir, and also, needs to be persistent in some way

Reply:
Yeah, it works for the current structure but when we get subsequent deploys working, we might want to tweak things a little. It's hard to test right now though. We can actually grab the latest dir persistently by getting the latest timestamped directory.